src/Atmos/Model/AtmosModel.jl

```diff
@@ -173,10 +178,33 @@ struct AtmosModel{
     tracers::TR
     "Large-scale forcing (Forcing information from GCMs, reanalyses, or observations)"
     lsforcing::LF
+    "Compressibility switch"
+    compressibility::C
     "Data Configuration (Helper field for experiment configuration)"
     data_config::DC
 end
 
+abstract type Compressibilty end
```

Comment:
@tapios if we change `Incompressible` to `Anelastic` / `SingleStack`, should `Compressibility` change?

Reply:
Spoke with @ilopezgp about this. Thanks!